mifospay/src/main/java/org/mifos/mobilewallet/mifospay/standinginstruction/ui/SIDetailsActivity.kt

```diff
@@ -68,6 +77,7 @@ class SIDetailsActivity : BaseActivity(), StandingInstructionContract.SIDetailsV
                     DatePickerDialog.OnDateSetListener { view, year, monthOfYear, dayOfMonth ->
                         tv_valid_till.text = "$dayOfMonth-${(monthOfYear + 1)}-$year"
                     }, year, month, day)
+            picker.datePicker.minDate=SimpleDateFormat("dd-MM-yyyy").parse(tv_valid_from.text.toString()).time
```

Comment:
Also always have a white space before and after the `=`. The Kotlin checks are currently not in place, so till the time they are, please try to properly format the code from your end only.

We are following [this](https://checkstyle.sourceforge.io/google_style.html) for our Java code. This can also be followed for `.kt` files. Do check it out

Reply:
oh my bad,ill correct it